int8 EarLeft
int8 EarRight
---
uint8 Data